spring-cloud-kubernetes-dependencies/pom.xml

```diff
@@ -35,8 +35,8 @@
 		<arquillian.version>1.6.0.Final</arquillian.version>
 		<arquillian-cube.version>1.15.2</arquillian-cube.version>
 		<hoverfly.version>0.13.0</hoverfly.version>
-		<kubernetes-client.version>5.5.0</kubernetes-client.version>
-		<kubernetes-java-client.version>13.0.0</kubernetes-java-client.version>
+		<kubernetes-fabric8-client.version>5.9.0</kubernetes-fabric8-client.version>
```

Comment:
I did two things here:

- rename the two clients properties to: `kubernetes-fabric8-client.version` and `kubernetes-native-client.version`. This is not really needed, but it is by far not the first time I stumble into this, a descriptive name would have saved me some good minutes. 
- I've upgraded to `5.9.0` to `5.5.0`